.travis.yml

```diff
@@ -4,8 +4,9 @@ os:
 #  - osx
 julia:
   - 0.6
+  - 0.7
   - nightly
 notifications:
   email: false
 after_success:
-  - julia -e 'cd(Pkg.dir("Showoff")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
+  - julia -e 'using Pkg; cd(Pkg.dir("Showoff")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())';
```

Comment:
is the `Pkg.dir` syntax not also deprecated in favor of `import Showoff; dirname(pathoff(Showoff))`?

Reply:
It is, but we're maintaining 0.6 support here I guess, so it'll have to stay as-is for the time being.